examples/Demo/Shared/Components/SiteSettingsPanel.razor

```diff
@@ -7,11 +7,10 @@
             <FluentSelect Id="color"
                           Items=@(Enum.GetValues<OfficeColor>())
                           OptionValue="@(c => c.ToAttributeValue())"
-                          OptionSelected="@(c => c == _selectedColorOption)"
+                          OptionSelected="@(c => c == ThemeService.SelectedAccentColor)"
                           TOption="OfficeColor"
                           Position="SelectPosition.Below"
-                          @bind-SelectedOption="@_selectedColorOption"
-                          @onchange="@(x => HandleColorChange(x))"
+                          @onchange="@(x => ThemeService.SetAccentColorAsync(x.Value!.ToString()!.GetEnumByDescription<OfficeColor>()))"
```

Comment:
Avoid to use an exclamation mark to get a property value. If the compilator say that this object can be null, please check the value :-) And don't use .ToString to convert an object to a string but `Convert.ToString`